CHANGELOG.md

```diff
@@ -4,12 +4,16 @@ Mapbox welcomes participation and contributions from everyone.
 # main
 
 # 10.8.0-rc.1
-## Features ✨ and improvements 🏁
-
 ## Bug fixes 🐞
 * Try recreate EGL surface when it throws exception. ([1589](https://github.com/mapbox/mapbox-maps-android/pull/1589))
 * Fix `MapboxMap` extension plugin functions throwing exceptions. ([1591](https://github.com/mapbox/mapbox-maps-android/pull/1591))
 * Fix concurrent modification exception when using widgets. ([1597](https://github.com/mapbox/mapbox-maps-android/pull/1597))
+* Relax LOD requirements for maps with insets and zero terrain exaggeration, leading to more content shown in easily readable map areas. ([1623](https://github.com/mapbox/mapbox-maps-android/pull/1623))
+* Make CameraManager.setCamera method exception free. In cases when incorrect CameraOptions are provided, error would be logged. ([1623](https://github.com/mapbox/mapbox-maps-android/pull/1623))
```

Comment:
@pengdev could we actually revert https://github.com/mapbox/mapbox-maps-android/pull/652 right here?

Reply:
let's follow up in a separate PR, as this is just bumping gl-native version.